Hummingbird.docc/HummingbirdWebSocket/HummingbirdWebSocket.md

```diff
@@ -8,9 +8,50 @@ Adds support for upgrading HTTP connections to WebSocket.
 
 ## Overview
 
-WebSockets is a protocol providing simultaneous two-way communication channels over a single TCP connection. Unlike HTTP where client requests are paired with a server response, WebSockets allow for communication in both directions asynchronously. It is designed to work over the HTTP ports 80 and 443 via an upgrade process where an initial HTTP request is sent before the connection is upgraded to a WebSocket connection.
+WebSockets is a protocol providing simultaneous two-way communication channels over a single TCP connection. Unlike HTTP where client requests are paired with a server response, WebSockets allow for communication in both directions asynchronously, for a prolonged period of time.
 
-HummingbirdWebSocket allows you to implement an HTTP1 server with WebSocket upgrade.
+It is designed to work over the HTTP ports 80 and 443 via an upgrade process where an initial HTTP request is sent before the connection is upgraded to a WebSocket connection.
+
+HummingbirdWebSocket allows you to implement an HTTP1 server with WebSocket upgrade. HummingbirdWebSocket is Autobahn compliant and supports both compression and TLS.
```

Comment:
Autobahn compliant does mean much to most people.

HummingbirdWebSocket passes all the tests in the  [Autobahn test suite](https://github.com/crossbario/autobahn-testsuite). 

Maybe?

Reply:
Sure